client/vitest.config.js

```diff
@@ -0,0 +1,14 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite'
+
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    plugins: [react()],
+    test: {
+        framework: 'vitest',
+        globals: true,
+        environment: 'jsdom'
+    }
+})
```

Comment:
best practice is to leave an empty line as the very last line of the file. Consider adding an empty line after this line